cmd/ethrex/cli.rs

```diff
@@ -384,3 +414,55 @@ pub async fn import_blocks(
     info!("Added {size} blocks to blockchain");
     Ok(())
 }
+
+pub async fn export_blocks(
+    path: &str,
+    data_dir: &str,
+    first_number: Option<u64>,
+    last_number: Option<u64>,
+) {
+    let data_dir = set_datadir(data_dir);
+    let store = open_store(&data_dir);
+    let start = first_number.unwrap_or_default();
+    // If we have no latest block then we don't have any blocks to export
+    let latest_number = match store.get_latest_block_number().await {
+        Ok(number) => number,
+        Err(StoreError::MissingLatestBlockNumber) => {
+            warn!("No blocks in the current chain, nothing to export!");
+            return;
+        }
+        Err(_) => panic!("Internal DB Error"),
+    };
+    // Check that the requested range doesn't exceed our current chain length
+    if last_number.is_some_and(|number| number > latest_number) {
+        warn!("The requested block range exceeds the current amount of blocks in the chain {latest_number}");
```

Comment:
```suggestion
        warn!("The requested block range exceeds the current amount of blocks in the chain: {latest_number}");
```
Should this be a `warn!` or an `error!`? Like, we are not exporting the blocks anyway so I don't know if an error would me more appropriate. Maybe it's not